apps/charterafrica/src/lib/tools/fetchGSheet.js

```diff
@@ -0,0 +1,52 @@
+import fetchJson from "@/charterafrica/utils/fetchJson";
+
+export async function fetchSpreadsheet(path, options) {
+  const params = {
+    ...options,
+    key: process.env.GOOGLE_API_KEY,
+  };
+  const res = await fetchJson.get(
+    `https://sheets.googleapis.com/v4/spreadsheets${path}`,
+    { params }
+  );
+  return res;
+}
+
+export const getSheetsPerSpreadsheet = async ({ spreadSheetId }) => {
+  const params = {
+    fields: "sheets.properties.title",
+  };
+  const d = await fetchSpreadsheet(`/${spreadSheetId}`, params);
+  const { sheets } = d;
+  if (sheets && sheets?.length) {
+    return sheets.map((item) => item?.properties?.title);
+  }
+  return [];
+};
+
+export const fetchSpreadsheetPerSheet = async ({
+  spreadSheetId,
+  sheetName,
+}) => {
+  const range = `${sheetName}!a:z`;
```

Comment:
Doesn't putting just the sheet name in range accomplish the same thing?
```suggestion
  const range = `${sheetName}`;
```